### ISSUE TITLE
NA Birds 3

### ISSUE THREAD

**Issue by samuelstevens** [samuelstevens/swin-transformer#3]
Dataset: NA Birds
Data: 100%
Model: Best iNat21 with hierarchy

Use a pre-trained swin-v2-base model pre-trained on iNat21 with hierarchy with the best validation loss, fine-tune it on NA Birds, then report the accuracy (and preferably attach training logs, link to WandB dashboards, etc).
